raiden/tests/mocked/test_matrix_transport.py

```diff
@@ -103,9 +108,137 @@ def mock_get_user_presence(self, user_id: str):
     monkeypatch.setattr(transport._raiden_service, "on_messages", mock_on_messages)
     monkeypatch.setattr(GMatrixClient, "get_user_presence", mock_get_user_presence)
 
+    mock_api_leave_room = lambda room_id: None
+
+    if hasattr(request, "param") and not request.param:
+
+        def raise_ex(room_id):
+            raise MatrixRequestError()
+
+        mock_api_leave_room = raise_ex
+
+    monkeypatch.setattr(transport._client.api, "leave_room", mock_api_leave_room)
+    monkeypatch.setattr(transport._client, "sync_token", "already_synced")
+
     return transport
 
 
+def create_new_users_for_node(mock_raiden_service=None, number_of_users=1):
+    users = list()
+    raiden_service = mock_raiden_service if mock_raiden_service else MockRaidenService()
```

Comment:
You don't need to instantiate a MockRaidenService if all you need is a signer.
There is the `raiden.tests.utils.factories.make_signer()` utility for that.